src/test/msgr/test_msgr.cc

```diff
@@ -323,7 +323,7 @@ TEST_P(MessengerTest, NameAddrTest) {
     client_msgr->get_mytype(), client_msgr->get_myaddrs());
   // Make should server_conn is the one we already accepted from client,
   // so it means client_msgr has the same addr when server connection has
-  ASSERT_EQ(1U, static_cast<Session*>(conn->get_priv().get())->get_count());
+  ASSERT_EQ(1U, static_cast<Session*>(server_conn->get_priv().get())->get_count());
```

Comment:
might want to s/Make should/verify that/ ? as the existing comment does not look right..